docs_input/api/stats/corr/cov.rst

```diff
@@ -5,6 +5,10 @@ cov
 
 Compute a covariance matrix
 
+.. note::
+   This function is currently not supported with host-based executors (CPU)
```

Comment:
Looking at the `cov` code it seems like it should be supported on the host, but just hasn't been updated yet. I don't see any functions that would prevent it from working.

Reply:
Yea, we just need to change the streams to executors